PR_Guide.md

```diff
@@ -0,0 +1,25 @@
+# PR Guidelines
+
+
+**Note**: This guide will act as a set of rules, that are required to be followed by anyone who is raising a PR in this project.
+
+# Key Points
+
+1. URL
+2. Assignee
+3. Label
+4. Elementary
+
+## URL
+Everyone should add the url of the ticket in his PR.
```

Comment:
rather than using "his" use "his/her", or even better "their" which will be gender independent